src/control/ros/RosTrajectoryExecutor.cpp

```diff
@@ -234,7 +234,11 @@ void RosTrajectoryExecutor::step(
 //==============================================================================
 void RosTrajectoryExecutor::abort()
```

Comment:
I'd prefer "cancel" instead of "abort". It seems like "cancel" and "abort" are different terms in ros_control; "cancel" is where the client requested to cancel the trajectory, and "abort" is where the server had to stop it for some other reason (e.g. force threshold has been violated).

http://wiki.ros.org/actionlib/DetailedDescription